README.md

```diff
@@ -26,7 +27,7 @@ segmentation map refers to pixels in the resized image, not the original input i
 | 6  | bus         | 13 | horse       | 20 | tv          |
 
 The model files are hosted on IBM Cloud Object Storage. The code in this repository deploys the model as a web service
-in a Docker container. This repository was developed as part of the [IBM Code Model Asset Exchange](https://developer.ibm.com/code/exchanges/models/).
+in a Docker container. This repository was developed as part of the [IBM Code Model Asset Exchange](https://developer.ibm.com/code/exchanges/models/) and the public API is powered by [IBM Cloud](https://ibm.biz/Bdz2XM).
```

Comment:
??

Reply:
@ptitzler This is part of the housekeeping change.